RULES.md

```diff
@@ -419,3 +420,12 @@ Even though `feed_info.start_date` and `feed_info.end_date` are optional, if one
 ### TooFastTravelNotice
 
 As implemented in the original [Google Python GTFS validator](https://github.com/google/transitfeed/wiki/FeedValidator), the calculated speed between stops should not be greater than 150 km/h (42 m/s SI or 93 mph). 
+
+<a name="AttributionWithoutRoleNotice"/>
+
+### AttributionWithoutRoleNotice
+
+At least one of the fields is_producer, is_operator, or is_authority should be set at 1.
+
+#### References:
+* [attributions.txt specification](https://gtfs.org/best-practices/#attributionsstxt)
```

Comment:
You can reference the spec instead of best practices here - https://gtfs.org/reference/static#attributionstxt.

Under `is_producer` it says:

>At least one of the fields is_producer, is_operator, or is_authority should be set at 1.